src/FortyTwo/Prompts/Confirm.hs

```diff
@@ -14,18 +15,18 @@ normalizeString :: String -> String
 normalizeString s = take 1 $ T.unpack $ T.strip . T.toLower $ T.pack s
 
 -- | Get a clean user input string
-getCleanConfirm :: IO String
-getCleanConfirm = do s <- getLine; return $ normalizeString s
+getCleanConfirm :: MonadIO m => m String
+getCleanConfirm = do s <- liftIO getLine; return $ normalizeString s
 
 -- | Ask a confirm falling back to a default value if no answer will be provided
-confirmWithDefault :: String -> Bool -> IO Bool
+confirmWithDefault :: MonadIO m => String -> Bool -> m Bool
 confirmWithDefault question defaultAnswer = do
```

Comment:
Can't we wrap the whole function in a `liftIO` call?

Reply:
Yeah, it makes no difference, really. I've wrapped the whole function.